src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -859,21 +877,54 @@ To get all submitted job IDs and names, use `list_submitted_jobs` pragma this wa
 #%list_submitted_jobs()
 ```
 
-==== Print results
+==== Export the workflow in xml format
```

Comment:
"XML" (capitals) would be the right wording, but I'm just being picky ;)

Reply:
Alright mister! xD